frontend/src/layouts/BaseLayout.vue

```diff
@@ -142,7 +142,11 @@
             <span class="dark-toggle text-caption q-ml-md">Advanced mode {{ advancedMode ? 'on' : 'off' }}</span>
             <span>
               <q-icon class="dark-toggle" right name="fas fa-question-circle">
-                <q-tooltip content-class="bg-muted dark-toggle shadow-2 q-pa-md" max-width="14rem">
+                <q-tooltip
+                  content-class="bg-muted dark-toggle shadow-2 q-pa-md"
+                  :hide-delay="$q.screen.lt.lg ? 1000 : 0"
```

Comment:
Why did you add a hide delay only on small screens? I'd guess it has to do with something related to clicking vs hovering to show the tooltip

Reply:
On mobile, you had to press and hold to see the tooltip, with this tweak you can click it and it hides after delay (Pressing and holding still work).